packages/sdk/src/HeadlessUniversalCheckout/Managers/PaymentMethodManagers/AchManager.ts

```diff
@@ -0,0 +1,158 @@
+import {
+    NativeEventEmitter,
+    NativeModules,
+    EmitterSubscription,
+} from 'react-native';
+import { PrimerComponentDataValidationError, PrimerInvalidComponentData, PrimerValidComponentData, PrimerValidatingComponentData } from 'src/models/PrimerComponentDataValidation';
+import { PrimerError } from 'src/models/PrimerError';
+import { EventType, eventTypes } from './Utils/EventType';
+import { AchStep, UserDetailsRetrieved } from 'src/models/ach/AchSteps';
+import { AchValidatableData } from 'src/models/ach/AchCollectableData';
+
+const { RNHeadlessUniversalCheckoutStripeAchUserDetailsComponent } = NativeModules;
+
+const eventEmitter = new NativeEventEmitter(RNHeadlessUniversalCheckoutStripeAchUserDetailsComponent);
+export interface AchManagerProps {
+    paymentMethodType: string;
+    onStep?: (data: AchStep) => void;
+    onError?: (error: PrimerError) => void;
+    onInvalid?: (data: PrimerInvalidComponentData<AchValidatableData>) => void;
+    onValid?: (data: PrimerValidComponentData<AchValidatableData>) => void;
+    onValidating?: (data: PrimerValidatingComponentData<AchValidatableData>) => void;
+    onValidationError?: (data: PrimerComponentDataValidationError<AchValidatableData>) => void;
+}
+
+export interface StripeAchUserDetailsComponent {
+    /**
+     * Starts the component, causing the {@link UserDetailsRetrieved} step
+     * to be emitted.
+     */
+    start(): Promise<void>;
+
+    /**
+     * Sets the customer's first name.
+     * @param value The customer's first name.
+     */
+    onSetFirstName(value: String): Promise<void>;
```

Comment:
[discussion]: `onSetFirstName` sounds like we'll receive some value - listener.
Can you sync with Web SDK engineer and ask about best practices?

Same for other methods here.

Reply:
I assume this it because of the 'on' prefix. In a way, it is a listener, as it could be bound to the first name input field. If we change it we'll be inconsistent, as this convention is already in use for for Klarna and Adyen, which are both live. 

@aabdaab, what do you think of this? Would 'onSetFirstName' be an appropriate name for a public API function that handles collection of the user's first name?

Reply:
@jnewc, @aabdaab, what do you think about using the `on` prefix for the component functions that update the collectable data?

Reply:
Handled as discussed [here](https://primerapi.slack.com/archives/C06U03QS6CU/p1722933705671819).